characteristic_read/web_app.js

```diff
@@ -0,0 +1,108 @@
+/**
+ *  Copyright 2020 Google LLC
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+var testDevice;
+
+const testServices = ['0b30acec-193e-11eb-adc1-0242ac120002'];
+
+const testCharacteristic = '0b30afd0-193e-11eb-adc1-0242ac120002';
+
+/**
+ * Load the device code to the Espruino IDE.
+ */
+function loadCode() {
+  fetch('device_code.js').then(response => response.text()).then(data => {
+    let url = 'http://www.espruino.com/webide?code=' + encodeURIComponent(data);
```

Comment:
We should eventually include the code loading mechanism directly in this test suite but this works for now.

Reply:
Sounds good. I'm not sure how yet, but I'll work on that the next change and integrate into following tests.